model/testing/src/icon4py/model/testing/pytest_config.py

```diff
@@ -156,3 +148,15 @@ def _get_grid(
             return grid_instance
         case _:
             return simple_grid.SimpleGrid()
+
+
+def pytest_runtest_setup(item):
+    backend = model_backends.BACKENDS[item.config.getoption("--backend")]
+    grid_option = item.config.getoption("--grid")
+    has_skip_values = grid_option is not None and grid_option != DEFAULT_GRID
```

Comment:
`--grid` has a default value, so I would suggest:
```suggestion
    has_skip_values = (grid_option != DEFAULT_GRID)
```

Reply:
I am actually not sure of the execution order so the default might not be set yet.